### PR TITLE
Use array_key_exists() instead of array_keys() and in_array()

### PR DESCRIPTION
**Pull request by ricordisamoa**
According to @thiemowmde at http://maettig.com/1397246220:
"never ever do `in_array( $key, array_keys( $array ) )`, that's just stupid"

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 require 'config.php';
 
 $lang = (isset($_GET['lang'])) ? $_GET['lang'] : 'en';
```
Comment (thiemowmde) [inline index.php:4]: Unrelated, but: http://tools.wmflabs.org/ws-cat-browser/?lang=%22%3E%3Ch1%3EWhoops%3Cscript%3Ealert%28%22BANG!%22%29%3C/script%3E%3C/h1%3E%3Ca%20href=%22http://disney.com/#

Comment (ricordisamoa) [inline index.php:4]: Oops! #5

**Comment by samwilson** [comment]
Hehe, yes! Perfectly stupid. :) I should have known being tired on a Friday night was not the time to release a new version. Thanks for the fix!

File: index.php (hunk 1 above)
Comment (samwilson) [inline index.php:4]: eek! fixed now. :-)

**Pull request merged**